src/mono/mono/mini/mini-windows.c

```diff
@@ -189,6 +189,7 @@ mono_runtime_install_handlers (void)
 	win32_seh_set_handler(SIGFPE, mono_sigfpe_signal_handler);
 	win32_seh_set_handler(SIGILL, mono_crashing_signal_handler);
 	win32_seh_set_handler(SIGSEGV, mono_sigsegv_signal_handler);
+	win32_seh_set_handler(SIGSEGV, mono_sigterm_signal_handler);
```

Comment:
SIGTERM ?

Reply:
D'Oh! What happens when you don't have a Windows environment to build this stuff. Changed.